mne/preprocessing/tests/test_pca_obs.py

```diff
@@ -0,0 +1,110 @@
+# Authors: The MNE-Python contributors.
+# License: BSD-3-Clause
+# Copyright the MNE-Python contributors.
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
```

Comment:
Shouldn't be imported at this level or test collection can fail for someone without Pandas installed
```suggestion
```